test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -1772,10 +1738,9 @@ function (Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthForm, Be
           return Expect.waitForSpyCall(test.successSpy, test);
         })
           .then(function (test) {
+            // it depends client to invoke `callGlobalSuccess`
+            // hence form is still disabling.
```

Comment:
would it make sense to wait for `callGlobalSuccess` ?

Reply:
it does wait for global success function at line above
`return Expect.waitForSpyCall(test.successSpy, test);`
My comment is a little misleading. I'm trying to explain why the form is still disabled even when `callGlobalSuccess` is invoked.

Reply:
OK, I see. Yes, I was confused by the comment. Perhaps the meaning can be clarified?

Reply:
yeah, updated comments.